MaybeGame/Player.cs

```diff
@@ -213,13 +217,10 @@ private int SetDefence(int stamina)
         }
         private double SetDodgeChance(int dexterity)
         {
-            var minDodgeChance = 0.00d;
-            var maxDodgeChance = 1.00d;
-            var dodgeChance = 0d;
+            var maxDodgeChance = 0.95d;
+            var dodgeChance = DodgeChance;
```

Comment:
Why do you need to create a variable to store data from property?

Reply:
Do not know, probably brain lag.